README.md

```diff
@@ -1,5 +1,8 @@
 #  AI-lab Software Templates
 
+> [!IMPORTANT]  
+> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, alongside their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see [the installer we have prepared](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```

Comment:
```suggestion
> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, along with their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see [the installer we have prepared](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```

I'm good with Fanis' nit, personally I think if the comma is there we break apart `alongside`, wdyt?

Reply:
Updated.